x-pack/plugins/cases/server/client/cases/find.test.ts

```diff
@@ -63,4 +63,23 @@ describe('find', () => {
       expect(call.caseOptions).not.toHaveProperty('rootSearchFields');
     });
   });
+
+  describe('throwErrorIfSearchFieldsAreInvalid', () => {
```

Comment:
nit: This text makes me think that there is a function called `throwErrorIfSearchFieldsAreInvalid` which does the validation. I think it is clearer if we say something like `searchFields validation` or similar.

Reply:
I called it like that because the bulk get tests in your PR follow the same naming logic 😛

Reply:
Renamed

Reply:
> I called it like that because the bulk get tests in your PR follow the same naming logic 😛

The `bulkGet` had a function called `throwErrorIfFieldsAreInvalid` this is why it was there. In my recent PR, where I removed the field validation, I forgot to rename it.